controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -223,7 +223,7 @@ func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Resul
 			sccType := r.storageClassClaim.Spec.Type
 			sccEncryptionMethod := r.storageClassClaim.Spec.EncryptionMethod
 			_, scIsFSType := existing.Parameters["fsName"]
-			scEncryptionMethod, scHasEncryptionMethod := existing.Parameters["encryptionMethod"]
+			scEncryptionMethod, scHasEncryptionMethod := existing.Parameters["encryptionKMSID"]
 
```

Comment:
A suggestion
Let's create constants for such parameter keys to avoid such confusion ?